packages/default/scss/spreadsheet/_layout.scss

```diff
@@ -488,6 +488,9 @@
         padding: 0;
         z-index: 60;
     }
+    .k-spreadsheet .k-selection-wrapper .k-spreadsheet-editor-button.k-spreadsheet-last-column {
```

Comment:
Do we need this selector, as far as I see, we do not override another strong selector?

Reply:
This whole file needs to be reformated